_topic_map.yml

```diff
@@ -245,6 +245,15 @@ Topics:
 - Name: Integrating with Email
   File: integrate-with-email
 ---
+Name: Telemetry
```

Comment:
I feel like Telemetry would go after things like upgrading or troubleshooting in the TOC (maybe right before support)?